mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -36,148 +40,140 @@ interface PasscodePromptProps {
+  const { isLight } = useThemeContext()
   return (
     <>
-      <ThemedTouchableOpacity
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
+      <ThemedTouchableOpacityV2
+        light={tailwind('bg-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-100')}
         onPress={() => props.onCancel(USER_CANCELED)}
-        style={tailwind('items-end pt-2 pr-2')}
+        style={[tailwind('items-end pt-6 pr-6'), { borderTopLeftRadius: 15, borderTopRightRadius: 15 }]}
         testID='cancel_authorization'
         disabled={[TransactionStatus.BLOCK, TransactionStatus.SIGNING].includes(props.status)}
       >
-        <ThemedIcon
-          dark={tailwind('text-white')}
-          light={tailwind('text-black')}
-          iconType='MaterialIcons'
-          name='close'
-          size={26}
-        />
-      </ThemedTouchableOpacity>
-
-      <ThemedView
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
+        <View
+          style={[tailwind(`w-6 h-6 flex justify-center items-center rounded-full ${isLight ? 'border-mono-light-v2-900' : 'border-mono-dark-v2-900'}`), { borderWidth: 1.5 }]}
+        >
+          <ThemedIcon
+            dark={tailwind('text-mono-dark-v2-900')}
+            light={tailwind('text-mono-light-v2-900')}
+            iconType='MaterialIcons'
+            name='close'
+            size={16}
+          />
+        </View>
+      </ThemedTouchableOpacityV2>
+      <ThemedViewV2
+        light={tailwind('bg-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-100')}
         style={tailwind('w-full flex-1 flex-col')}
       >
+        <ThemedViewV2
+          light={tailwind('bg-mono-light-v2-100')}
+          dark={tailwind('bg-mono-dark-v2-100')}
+        >
+          <ThemedTextV2
+            style={[tailwind('text-center font-normal-v2 px-16 pt-3.5'), { marginBottom: props.status !== TransactionStatus.SIGNING && props.status !== TransactionStatus.AUTHORIZED ? 76 : 0 }]}
+          >
+            {props.title}
+          </ThemedTextV2>
+          {props.status === TransactionStatus.SIGNING && <ThemedActivityIndicatorV2 style={tailwind('py-4 my-1.5')} />}
+          {props.status === TransactionStatus.AUTHORIZED && <SuccessIndicator />}
+          <PinTextInputV2
+            cellCount={props.pinLength}
+            onChange={(pin) => {
+                  props.onPinInput(pin)
+                }}
+            testID='pin_authorize'
+            value={props.pin}
+          />
+          <View style={tailwind('px-8 text-sm text-center mb-14 mt-4')}>
+            {// show pin success message
+              props.status === TransactionStatus.AUTHORIZED &&
+                <ThemedTextV2
+                  testID='txn_authorization_message'
+                  dark={tailwind('text-mono-dark-v2-700')}
+                  light={tailwind('text-mono-light-v2-700')}
+                  style={tailwind('px-8 text-sm text-center font-normal-v2')}
```

Comment:
here padding is 20 px only as per figma

Reply:
But we need to position the text in center. It is been used across all the other screens

Reply:
you can use combination of margin like `mb-16` and on pin code add `mt-3`

Reply:
@siradji plz check this